test-data/unit/fine-grained-modules.test

```diff
@@ -751,15 +751,16 @@ main:1: note: (Perhaps setting MYPYPATH or using the "--ignore-missing-imports"
 main:2: error: Incompatible types in assignment (expression has type "int", variable has type "str")
 
 [case testAddFileWhichImportsLibModuleWithErrors]
+# flags: --no-silence-site-packages
```

Comment:
Why is this necessary here? Is `broken` installed as a PEP 561 package?

Reply:
Since opening this, I also implemented silencing typeshed, which `broken` is simulated as being part of via `alt_lib_path`.